eqs/src/eth_polling.rs

```diff
@@ -365,6 +370,19 @@ impl EthPolling {
                     updated_state.last_reported_index = Some(current_index);
                     self.last_event_index = Some(current_index);
 
+                    //add assets to local map
+                    for wrap in wraps {
```

Comment:
It would be better to update this mapping on sponsor, rather than wrap, since sponsor happens exactly once per asset and it's the operation that actually establishes the mapping. Currently, though, there is no event for sponsors. Since we're going to be redeploying the contract soon anyway (to handle 128 bit amounts) should we also add an event for sponsors? cc @philippecamacho  @sveitser

Reply:
I agree, I created a [ticket](https://github.com/EspressoSystems/cape/issues/1068) so that we can come back to it later.